### PR TITLE
Terms

### PR DESCRIPTION
**Pull request by JosephKTsai**
Adds the terms and conditions page, in addition to linking to the terms and conditions page from the home, create meeting, and conduct meeting pages.

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const routes = require('next-routes')();
 // @RANStartRoutes
 
 routes.add('meeting', '/meeting/:meetingId');
+routes.add('terms', '/terms');
```
Comment (bbasinsk) [inline routes.js]: This actually isn't needed! https://nextjs.org/learn/basics/navigate-between-pages

**Review by bbasinsk — COMMENTED** [review]
Could you attach a screenshot? And remove `package-lock.json` from 'Files changed'?

**Comment by JosephKTsai** [comment]
Here's an example of what the terms and conditions looks like within the footer of the page:

Screen Shot 2019-05-15 at 8 55 41 PM

**Comment by JosephKTsai** [comment]
Screenshots of Terms and Conditions along with screenshots of the link at the bottom of the page

Screen Shot 2019-05-15 at 9 22 22 PM
Screen Shot 2019-05-15 at 9 22 25 PM
Screen Shot 2019-05-15 at 8 55 41 PM
Screen Shot 2019-05-15 at 9 23 06 PM
Screen Shot 2019-05-15 at 9 23 33 PM

**Review by bbasinsk — APPROVED** [review]

**Pull request merged**